From: Boris BREZILLON <boris.brezillon@free-electrons.com>
Date: Fri, 3 Nov 2017 17:52:01 +0100
Subject: crypto: marvell - Add a NULL entry at the end of
 mv_cesa_plat_id_table[]

Git-commit: 9c9034464860707669e91ce367c9eeb65e4175cd
Patch-mainline: v4.15-rc1
References: fate#326470

struct platform_device_id should be NULL terminated to let the core detect
where the last entry is.

Fixes: 07c50a8be41a ("crypto: marvell - Add a platform_device_id table")
Signed-off-by: Boris Brezillon <boris.brezillon@free-electrons.com>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
Signed-off-by: Mian Yousaf Kaukab <yousaf.kaukab@suse.com>
---
 drivers/crypto/marvell/cesa.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/crypto/marvell/cesa.c b/drivers/crypto/marvell/cesa.c
index 2c0d6bd94165..293832488cc9 100644
--- a/drivers/crypto/marvell/cesa.c
+++ b/drivers/crypto/marvell/cesa.c
@@ -591,6 +591,7 @@ static int mv_cesa_remove(struct platform_device *pdev)
 
 static const struct platform_device_id mv_cesa_plat_id_table[] = {
 	{ .name = "mv_crypto" },
+	{ /* sentinel */ },
 };
 MODULE_DEVICE_TABLE(platform, mv_cesa_plat_id_table);
 
-- 
2.11.0

